source/connect/tls.txt

```diff
@@ -0,0 +1,241 @@
+and will issue an error if used.
+
+Setting ``tlsDisableOCSPEndpointCheck`` and ``tlsDisableCertificateRevocationCheck`` has no effect.
+
+The Online Certificate Status Protocol (OCSP) (see `RFC 6960 <https://tools.ietf.org/html/rfc6960>`__) is partially supported
+with the following notes:
+
+- The Must-Staple extension (see `RFC 7633 <https://tools.ietf.org/html/rfc7633>`__) is ignored.
+- Connection will continue if a Must-Staple certificate is presented without a stapled response and the OCSP responder is down.
+- Connection will not continue if the client receives a revoked response from an OCSP responder.
+
+Native TLS Support on Windows (Secure Channel)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The MongoDB C Driver supports the Windows native TLS library (Secure Channel, or SChannel) and its native crypto library
+(Cryptography API: Next Generation, or CNG).
+
+When compiled against the Windows native libraries, the ``ca_dir`` option of a `mongoc_ssl_opt_t <{+api-libmongoc+}/mongoc_ssl_opt_t.html>`__ is not supported
+and will issue an error if used.
+
+Encrypted PEM files (e.g., setting ``tlsCertificateKeyPassword``) are also not supported and will result in error when
+attempting to load them.
+
+When ``tlsCAFile`` is set, the driver will only allow server certificates issued by the authority (or authorities) provided.
+When no ``tlsCAFile`` is set, the driver will look up the Certificate Authority using the ``System Local Machine Root``
+certificate store to confirm the provided certificate.
+
+When ``crl_file`` is set with `mongoc_ssl_opt_t <{+api-libmongoc+}/mongoc_ssl_opt_t.html>`__, the driver will import the revocation list to the
+``System Local Machine Root`` certificate store.
+
+Setting ``tlsDisableOCSPEndpointCheck`` has no effect.
+
+The Online Certificate Status Protocol (OCSP) (see `RFC 6960 <https://tools.ietf.org/html/rfc6960>`__) is partially supported
+with the following notes:
+
+- The Must-Staple extension (see `RFC 7633 <https://tools.ietf.org/html/rfc7633>`__) is ignored.
+- Connection will continue if a Must-Staple certificate is presented without a stapled response and the OCSP responder is down.
+- Connection will not continue if the client receives a revoked response from an OCSP responder.
+- When a ``crl_file`` is set with `mongoc_ssl_opt_t <{+api-libmongoc+}/mongoc_ssl_opt_t.html>`__, and the ``crl_file`` revokes the server's certificate, the OCSP response
+  takes precedence. For example, if the server presents a certificate with a valid stapled OCSP response, the certificate is considered
+  valid even if the ``crl_file`` marks it as revoked.
+
+.. _secure-transport:
+
+Native TLS Support on macOS / Darwin (Secure Transport)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The MongoDB C Driver supports the Darwin (OS X, macOS, iOS) native TLS library (Secure Transport), and its native crypto
```

Comment:
S: I'm a little confused by the parentheses here; I think you can remove the "(OS X, macOS, iOS)" one, maybe explain briefly how "native TLS library relates" to "(Secure Transport)", and then just say "and Common Crypto, its native crypto library."